### ISSUE TITLE
`ObjectMouseOver` doesn't work after triggering `SelectedChange` 

### ISSUE THREAD

**Issue by DevArt002** [jscastro76/threebox#237]
Hi. 
I am going to change cursor when tb object gets hovered. (Not pointer cursor) I added the change functionality under the **ObjectMouseOver** eventlistener. It works well until tb object is clicked. But once I click the tb object, the **ObjectMouseOver** eventlistener doesn't work. Maybe, tb object **SelectedChange** eventlistener blocks **ObjectMouseOver**

I shared the code snippet:
```
obj.addEventListener(
  'SelectedChange',
  () => {
    // TODO select
  },
  false,
);
obj.addEventListener(
  'ObjectMouseOver',
  () => {
    // TODO mouse over
    this.map.getCanvasContainer().style.cursor = 'not-allowed';
  },
  false,
);
```

**Comment by jscastro76** [comment]
Let me check it out

**Comment by DevArt002** [comment]
Thank you @jscastro76 !

**Comment by jscastro76** [comment]
You're totally right, "ObjectMouseOver" is only fired if it's not the object selected... my fault surely when combining the different logics for objects/extrusions. I have just solved locally, I'll push the change today to the repo as the first issue of v2.2.3. But it won't be available as module until end of this week

**Comment by DevArt002** [comment]
Sounds great! Thanks for letting me know. I can't wait to see the new version soon!

**Comment by jscastro76** [comment]
Available for code testing and bundle use, not available yet as module until v2.2.3 is uploaded to npm
@DevArt002 thanks for detecting it and provide feedback
